.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+    - name: Ask a Question
+      url: https://github.com/IBM/the-mesh-for-data/discussions/new?category=help
+      about: Ask the community for help
+    - name: Feature Request
+      url: https://github.com/IBM/the-mesh-for-data/discussions/new?category=ideas
+      about: Share ideas for new features
+    - name: Bug Report
+      url: https://github.com/IBM/the-mesh-for-data/issues/new
+      about: Report a reproducable bug
```

Comment:
Wouldn't the 3rd option bring us to the same place?

Reply:
No